pkg/process/metadata/parser/service_windows_test.go

```diff
@@ -56,16 +56,15 @@ func TestWindowsExtractServiceMetadata(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			mockConfig := ddconfig.MockSystemProbe(t)
-			mockConfig.SetWithoutSource("system_probe_config.process_service_inference.enabled", true)
-
+			enabled := true
+			useWindowsServiceName := true
```

Comment:
Should be false

Reply:
enabled should be true, and useWindowsServiceName should be false in this case.